core/src/appleMain/kotlin/ScannerBuilder.kt

```diff
@@ -17,18 +19,39 @@ public actual class ScannerBuilder {
 
     public actual var filters: List<Filter>? = null
 
-    public var scanOptions: AppleScanner.ScanOptions? = null
+    /**
+     * Specifies whether the scan should run without duplicate filtering. This corresponds to
+     * Apple's CBCentralManagerScanOptionAllowDuplicatesKey scanning option.
```

Comment:
nitpick:
- "Core Bluetooth" is slightly more specific 🤷 
- brackets around the constants make them clickable in IDE (and might allow Dokka to link them in the generated KDoc documentation)

```suggestion
     * Core Bluetooth's [CBCentralManagerScanOptionAllowDuplicatesKey] scanning option.
```